dev-server/dist/index.html

```diff
@@ -1,32 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>react-json-view</title>
-    <meta charset="UTF-8">
-    <!--[if lt IE 9]>
-    <script>
-    (function(){
-      var ef = function(){};
-      window.console = window.console || {log:ef,warn:ef,error:ef,dir:ef};
-    }());
-    </script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv-printshiv.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/3.4.0/es5-shim.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/3.4.0/es5-sham.js"></script>
-    <![endif]-->
-    <script src="https://unpkg.com/react@15/dist/react.js"></script>
-    <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>
-    <style>
-      .react-json-view {
-        padding: 4px 6px;
-        border-radius: 3px;
-        /*display: inline-block;*/
-      }
-    </style>
-  </head>
-  <body>
-  <div id="app-container"></div>
-  <script type="text/javascript" src="./main.js"></script>
-  </body>
-</html>
\ No newline at end of file
+    <head>
```

Comment:
It looks like your editor has a white space of 4 set, instead of 2. I would try to see if you can set it to 2 for this repo and re-save this file to be consistent with the existing styles for this project.

Reply:
Done